modules/web/src/app/shared/components/application-list/component.ts

```diff
@@ -137,7 +137,9 @@ export class ApplicationListComponent implements OnInit, OnDestroy {
         this.applications = [];
```

Comment:
Getting "Expression has changed error"

<img width="1244" alt="expression-change" src="https://user-images.githubusercontent.com/17727069/215975852-59ba15d4-118c-4d48-b4f4-7a51cd081d36.png">

Reply:
This issue is already present in dev so it should be fixed in a separate ticket.